### ISSUE TITLE
Windows installation can't deploy traefik-forward-auth

### ISSUE THREAD

**Issue by kkaempf** [epinio/epinio#674]
`epinio install` fails on Windows with

```
panic: Deployment traefik-forward-auth failed with error: create Traefik Forward Auth Middleware: the server could not find the requested resource


goroutine 57 [running]:
github.com/epinio/epinio/internal/cli/clients.(*InstallClient).Install.func1(0xc000451fb0, 0x220de70, 0xc00009e010, 0x2218b30, 0xc0000e6780, 0x221d628, 0xc00081d120, 0xc0004d27d4)
        /abuild/projects/rancher/epinio/epinio/internal/cli/clients/install_client.go:168 +0x23f
created by github.com/epinio/epinio/internal/cli/clients.(*InstallClient).Install
        /abuild/projects/rancher/epinio/epinio/internal/cli/clients/install_client.go:165 +0x10e5
```

**Comment by andreas-kupries** [comment]
This is not an issue in the epinio client itself, right ? "Just" in the traefik it deploys ?

**Comment by kkaempf** [comment]
How can I judge this ? I only observe a "panic" during the install.

**Comment by andreas-kupries** [comment]
Setting the environment variable `DEBUG=true` will allow you to observe the low-level `kubectl`, `helm`, etc. commands run by `epinio` under the hood. Together `TRACE_LEVEL=6` for a bit of overall view on where the code is and we should be able to better see what is going on.

The good thing is that Traefik is one of the deployments not run concurrently with anything else, i.e. we do not to de-tangle the output from different deployments.

**Comment by kkaempf** [comment]
Trace output
```
2021/08/12 07:51:56 InstallClient/Install "level"=1 "msg"="start"
2021/08/12 07:51:56 InstallClient/Install "level"=2 "msg"="process cli options"
2021/08/12 07:51:56 InstallClient/Install "level"=2 "msg"="fill defaults into options"
2021/08/12 07:51:56 InstallClient/Install "level"=2 "msg"="show option configuration"
2021/08/12 07:51:56 InstallClient/Install "level"=2 "msg"="deploy"  "Deployment"="linkerd"
2021/08/12 07:51:56 InstallClient/Install/Deploy "level"=3 "msg"="start"
2021/08/12 07:51:56 InstallClient/Install/Deploy "level"=3 "msg"="check presence, local service"
2021/08/12 07:51:56 InstallClient/Install/Deploy "level"=3 "msg"="check presence, linkerd namespace"
2021/08/12 07:51:56 InstallClient/Install/Deploy "level"=3 "msg"="deploying linkerd"
2021/08/12 07:52:26 InstallClient/Install/Deploy "level"=3 "msg"="return"
2021/08/12 07:52:26 InstallClient/Install "level"=2 "msg"="return"
2021/08/12 07:52:26 InstallClient/Install "level"=2 "msg"="deploy"  "Deployment"="traefik"
2021/08/12 07:52:26 InstallClient/Install/Deploy "level"=3 "msg"="start"
2021/08/12 07:52:26 InstallClient/Install/Deploy "level"=3 "msg"="config"  "skipTraefik"=false
2021/08/12 07:52:26 InstallClient/Install/Deploy "level"=3 "msg"="check presence, local service"
2021/08/12 07:52:26 InstallClient/Install/Deploy "level"=3 "msg"="check presence, system service"
2021/08/12 07:52:26 InstallClient/Install/Deploy "level"=3 "msg"="service present"
2021/08/12 07:52:26 InstallClient/Install/Deploy "level"=3 "msg"="return"
2021/08/12 07:52:26 InstallClient/Install "level"=2 "msg"="return"
2021/08/12 07:52:26 InstallClient/Install "level"=2 "msg"="ensure system-domain"
2021/08/12 07:52:26 InstallClient/Install "level"=2 "msg"="check"  "Deployment"="kubed"
2021/08/12 07:52:26 InstallClient/Install "level"=2 "msg"="return"
2021/08/12 07:52:26 InstallClient/Install "level"=2 "msg"="check"  "Deployment"="cert-manager"
2021/08/12 07:52:26 InstallClient/Install "level"=2 "msg"="return"
2021/08/12 07:52:26 InstallClient/Install "level"=2 "msg"="check"  "Deployment"="dex"
2021/08/12 07:52:26 InstallClient/Install "level"=2 "msg"="return"
2021/08/12 07:52:26 InstallClient/Install "level"=2 "msg"="check"  "Deployment"="epinio"
2021/08/12 07:52:26 InstallClient/Install "level"=2 "msg"="return"
2021/08/12 07:52:26 InstallClient/Install "level"=2 "msg"="check"  "Deployment"="gitea"
2021/08/12 07:52:26 InstallClient/Install "level"=2 "msg"="return"
2021/08/12 07:52:26 InstallClient/Install "level"=2 "msg"="check"  "Deployment"="epinio-registry"
2021/08/12 07:52:26 InstallClient/Install "level"=2 "msg"="return"
2021/08/12 07:52:26 InstallClient/Install "level"=2 "msg"="check"  "Deployment"="tekton"
2021/08/12 07:52:26 InstallClient/Install "level"=2 "msg"="return"
2021/08/12 07:52:26 InstallClient/Install "level"=2 "msg"="check"  "Deployment"="service-catalog"
2021/08/12 07:52:26 InstallClient/Install "level"=2 "msg"="return"
2021/08/12 07:52:26 InstallClient/Install "level"=2 "msg"="check"  "Deployment"="traefik-forward-auth"
2021/08/12 07:52:26 InstallClient/Install "level"=2 "msg"="return"
2021/08/12 07:52:26 InstallClient/Install "level"=2 "msg"="deploy"  "Deployment"="kubed"
2021/08/12 07:52:26 InstallClient/Install "level"=2 "msg"="deploy"  "Deployment"="cert-manager"
2021/08/12 07:52:26 InstallClient/Install "level"=2 "msg"="deploy"  "Deployment"="traefik-forward-auth"
2021/08/12 07:52:26 InstallClient/Install "level"=2 "msg"="deploy"  "Deployment"="tekton"
2021/08/12 07:52:26 InstallClient/Install "level"=2 "msg"="deploy"  "Deployment"="epinio"
2021/08/12 07:52:26 InstallClient/Install "level"=2 "msg"="deploy"  "Deployment"="gitea"
2021/08/12 07:52:26 InstallClient/Install "level"=2 "msg"="deploy"  "Deployment"="dex"
2021/08/12 07:52:26 InstallClient/Install "level"=2 "msg"="deploy"  "Deployment"="service-catalog"
2021/08/12 07:52:26 InstallClient/Install "level"=2 "msg"="deploy"  "Deployment"="epinio-registry"
2021/08/12 07:52:26 InstallClient/Install/Deploy "level"=3 "msg"="start"
2021/08/12 07:52:26 InstallClient/Install/Deploy "level"=3 "msg"="start"
2021/08/12 07:52:27 InstallClient/Install/Deploy "level"=3 "msg"="creating namespace"  "namespace"="epinio-registry"
I0812 07:52:28.111376   12988 request.go:655] Throttling request took 1.1655161s, request: GET:https://172.20.140.55:6443/api/v1/namespaces/cert-manager
2021/08/12 07:52:28 InstallClient/Install/Deploy "level"=3 "msg"="creating namespace"  "namespace"="cert-manager"
2021/08/12 07:52:28 InstallClient/Install/Deploy "level"=3 "msg"="extracting chart file"  "name"="container-registry-0.1.0.tgz"
2021/08/12 07:52:28 InstallClient/Install/Deploy "level"=3 "msg"="local transient tar archive"  "name"="C:\\Users\\blubb\\AppData\\Local\\Temp\\epinio808585855"
2021/08/12 07:52:29 InstallClient/Install/Deploy "level"=3 "msg"="htpasswd from credentials"  "htpasswd"="f2f1ab2e6a0827d8:$2a$10$9JuEBtw2MV4P4WnDdoaEduDowS1ogJ6wjvUeb5kpfOnjIeDCO2f7K"
2021/08/12 07:52:29 InstallClient/Install/Deploy "level"=3 "msg"="system domain"  "domain"="172.20.140.55.omg.howdoi.website"
2021/08/12 07:52:29 InstallClient/Install/Deploy "level"=3 "msg"="create properly annotated secret"
2021/08/12 07:52:29 InstallClient/Install/Deploy "level"=3 "msg"="extracting chart file"  "name"="cert-manager-v1.2.0.tgz"
2021/08/12 07:52:29 InstallClient/Install/Deploy "level"=3 "msg"="local transient tar archive"  "name"="C:\\Users\\blubb\\AppData\\Local\\Temp\\epinio370564361"
2021/08/12 07:52:29 InstallClient/Install/Deploy "level"=3 "msg"="assembling helm command"
2021/08/12 07:52:29 InstallClient/Install/Deploy "level"=3 "msg"="assembled helm command"  "command"="helm install cert-manager --namespace cert-manager C:\\Users\\blubb\\AppData\\Local\\Temp\\epinio370564361 --set installCRDs=true --set extraArgs[0]=--enable-certificate-owner-ref=true"
2021/08/12 07:52:29 InstallClient/Install/Deploy "level"=3 "msg"="run helm command"
2021/08/12 07:52:29 InstallClient/Install/Deploy "level"=3 "msg"="waiting for cert manager to be present and active"
2021/08/12 07:52:32 InstallClient/Install "level"=2 "msg"="return"
2021/08/12 07:52:33 InstallClient/Install/Deploy "level"=3 "msg"="completed helm command"
2021/08/12 07:52:33 InstallClient/Install/Deploy "level"=3 "msg"="waiting for pods to exist, and run"
2021/08/12 07:52:54 InstallClient/Install/Deploy "level"=3 "msg"="create cluster issuer"  "issuer"="{\n\t\"apiVersion\": \"cert-manager.io/v1alpha2\",\n\t\"kind\": \"ClusterIssuer\",\n\t\"metadata\": {\n\t\t\"name\": \"letsencrypt-production\"\n\t},\n\t\"spec\": {\n\t\t\"acme\" : {\n\t\t\t\"email\" : \"epinio@suse.com\",\n\t\t\t\"server\" : \"https://acme-v02.api.letsencrypt.org/directory\",\n\t\t\t\"privateKeySecretRef\" : {\n\t\t\t\t\"name\" : \"letsencrypt-production\"\n\t\t\t},\n\t\t\t\"solvers\" : [\n\t\t\t{\n\t\t\t\t\"http01\" : {\n\t\t\t\t\t\"ingress\" : {\n\t\t\t\t\t\t\"class\" : \"traefik\"\n\t\t\t\t\t}\n\t\t\t\t}\n\t\t\t}\n\t\t\t]\n\t\t}\n\t}\n}"
2021/08/12 07:52:57 InstallClient/Install/Deploy "level"=3 "msg"="create cluster issuer"  "issuer"="{\n\t\"apiVersion\": \"cert-manager.io/v1alpha2\",\n\t\"kind\": \"ClusterIssuer\",\n\t\"metadata\": {\n\t\t\"name\": \"selfsigned-issuer\"\n\t},\n\t\"spec\": {\n\t\t\"selfSigned\" : {}\n\t}\n}"
2021/08/12 07:52:57 InstallClient/Install/Deploy "level"=3 "msg"="create private CA cert"  "spec"="{\n\t\t\"apiVersion\" : \"cert-manager.io/v1alpha2\",\n\t\t\"kind\"       : \"Certificate\",\n\t\t\"metadata\"   : {\n\t\t\t\"name\" : \"epinio-ca\"\n\t\t},\n\t\t\"spec\" : {\n\t\t\t\"isCA\"       : true,\n\t\t\t\"commonName\" : \"epinio-ca\",\n\t\t\t\"secretName\" : \"epinio-ca-root\",\n\t\t\t\"privateKey\" : {\n\t\t\t\t\"algorithm\" : \"ECDSA\",\n\t\t\t\t\"size\"      : 256\n\t\t\t},\n\t\t\t\"issuerRef\" : {\n\t\t\t\t\"name\" : \"selfsigned-issuer\",\n\t\t\t\t\"kind\" : \"ClusterIssuer\"\n\t\t\t}\n\t\t}\n\t}"
2021/08/12 07:52:57 InstallClient/Install/Deploy "level"=3 "msg"="create cluster issuer"  "issuer"="{\n\t\"apiVersion\" : \"cert-manager.io/v1alpha2\",\n\t\"kind\"       : \"ClusterIssuer\",\n\t\"metadata\"   : {\n\t\t\"name\" : \"epinio-ca\"\n\t},\n\t\"spec\" : {\n\t\t\"ca\" : {\n\t\t\t\"secretName\": \"epinio-ca-root\"\n\t\t}\n\t}\n}"
2021/08/12 07:52:57 InstallClient/Install/Deploy "level"=3 "msg"="apply done"
2021/08/12 07:52:57 InstallClient/Install/Deploy "level"=3 "msg"="return"
2021/08/12 07:52:57 InstallClient/Install "level"=2 "msg"="return"
2021/08/12 07:52:58 InstallClient/Install/Deploy "level"=3 "msg"="issue registry cert"
2021/08/12 07:52:58 InstallClient/Install/Deploy "level"=3 "msg"="assembling helm command"
2021/08/12 07:52:58 InstallClient/Install/Deploy "level"=3 "msg"="assembled helm command"  "command"="helm install epinio-registry --namespace epinio-registry C:\\Users\\blubb\\AppData\\Local\\Temp\\epinio808585855 --set auth.htpasswd=f2f1ab2e6a0827d8:$2a$10$9JuEBtw2MV4P4WnDdoaEduDowS1ogJ6wjvUeb5kpfOnjIeDCO2f7K --set domain=epinio-registry.172.20.140.55.omg.howdoi.website --set createNodePort=true"
2021/08/12 07:52:58 InstallClient/Install/Deploy "level"=3 "msg"="run helm command"
2021/08/12 07:52:59 InstallClient/Install/Deploy "level"=3 "msg"="completed helm command"
2021/08/12 07:52:59 InstallClient/Install/Deploy "level"=3 "msg"="waiting for pods to exist"
2021/08/12 07:52:59 InstallClient/Install/Deploy "level"=3 "msg"="waiting for pods to run"
I0812 07:53:03.117358   12988 request.go:655] Throttling request took 1.0333309s, request: GET:https://172.20.140.55:6443/api/v1/namespaces/epinio/secrets/epinio-tls
2021/08/12 07:53:07 InstallClient/Install "level"=2 "msg"="return"
2021/08/12 07:53:12 InstallClient/Install/Deploy "level"=3 "msg"="return"
2021/08/12 07:53:12 InstallClient/Install "level"=2 "msg"="return"
2021/08/12 07:53:12 InstallClient/Install "level"=2 "msg"="return"
2021/08/12 07:53:18 InstallClient/Install "level"=2 "msg"="return"
2021/08/12 07:53:59 InstallClient/Install "level"=2 "msg"="return"
2021/08/12 07:54:00 InstallClient/Install "level"=2 "msg"="return"
panic: Deployment traefik-forward-auth failed with error: create Traefik Forward Auth Middleware: the server could not find the requested resource


goroutine 60 [running]:
github.com/epinio/epinio/internal/cli/clients.(*InstallClient).Install.func1(0xc0004f0cf0, 0x1ffde70, 0xc000034060, 0x2008b30, 0xc000630500, 0x200d628, 0xc0001076c0, 0xc0005ecfa0)
        /abuild/projects/rancher/epinio/epinio/internal/cli/clients/install_client.go:168 +0x23f
created by github.com/epinio/epinio/internal/cli/clients.(*InstallClient).Install
        /abuild/projects/rancher/epinio/epinio/internal/cli/clients/install_client.go:165 +0x10e5
```

**Comment by kkaempf** [comment]
Debug output
```

­ƒÜó Epinio installing...

Configuration...
  ­ƒº¡  skip-traefik: false
  ­ƒº¡  skip-linkerd: false
  ­ƒº¡  skip-cert-manager: false
  ­ƒº¡  system_domain:
  ­ƒº¡  email_address: epinio@suse.com
  ­ƒº¡  user: 36dbeb50389c2851
  ­ƒº¡  password: 2ca2d97ec513b38a
  ­ƒº¡  tls-issuer: epinio-ca
  ­ƒº¡  use-internal-registry-node-port: true
  ­ƒº¡  ingress-service-ip:

­ƒÜó Deploying Linkerd...Executing: kubectl [apply --filename C:\Users\blubb\AppData\Local\Temp\epinio936627703] (in: C:\Users\blubb\Downloads)
Executing: kubectl [apply --filename C:\Users\blubb\AppData\Local\Temp\epinio681065194] (in: C:\Users\blubb\Downloads)

Ô£ö´©Å  Linkerd deployed

ÔÜá´©Å  System Traefik Ingress present, skipping traefik installation and related flags `--ingress-service-ip`  will be ignored
.
Ô£ö´©Å  Using system_domain: 172.20.140.55.omg.howdoi.website

­ƒÜó Deploying TraefikForwardAuth ...
­ƒÜó Deploying Kubed...
­ƒÜó Deploying Tekton...
­ƒÜó Deploying Epinio...
­ƒÜó Deploying Dex...
­ƒÜó Deploying Gitea...Executing: helm [install kubed --namespace kubed C:\Users\blubb\AppData\Local\Temp\epinio236636993] (in: C:\Users\blubb\Downloads)
Executing: kubectl [apply --namespace epinio --filename C:\Users\blubb\AppData\Local\Temp\epinio796996524] (in: C:\Users\blubb\Downloads)

­ƒÜó Deploying Registry...
­ƒÜó Deploying ServiceCatalog...Executing: kubectl [apply --namespace epinio --filename C:\Users\blubb\AppData\Local\Temp\epinio553433022] (in: C:\Users\blubb\Downloads)

­ƒÜó Deploying CertManager....Executing: helm [install gitea --values C:\Users\blubb\AppData\Local\Temp\epinio557868037 --namespace gitea https://dl.gitea.io/charts/gitea-2.1.3.tgz] (in: C:\Users\blubb\Downloads)
..Executing: kubectl [apply --filename C:\Users\blubb\AppData\Local\Temp\epinio702429088] (in: C:\Users\blubb\Downloads)
Executing: helm [install service-catalog --namespace service-catalog C:\Users\blubb\AppData\Local\Temp\epinio777617874] (in: C:\Users\blubb\Downloads)
.Executing: helm [install cert-manager --namespace cert-manager C:\Users\blubb\AppData\Local\Temp\epinio370564361 --set installCRDs=true --set extraArgs[0]=--enable-certificate-owner-ref=true] (in: C:\Users\blubb\Downloads)
..Executing: kubectl [apply --namespace tekton-staging --filename C:\Users\blubb\AppData\Local\Temp\epinio727151828] (in: C:\Users\blubb\Downloads)
.Executing: kubectl [apply --filename C:\Users\blubb\AppData\Local\Temp\epinio345721379] (in: C:\Users\blubb\Downloads)
..................
Ô£ö´©Å  Kubed deployed
................................................................................................................................................Executing: kubectl [wait --for condition=established --timeout 300s crd/clustertasks.tekton.dev] (in: C:\Users\blubb\Downloads)
.Executing: kubectl [wait --for condition=established --timeout 300s crd/conditions.tekton.dev] (in: C:\Users\blubb\Downloads)
...Executing: kubectl [wait --for condition=established --timeout 300s crd/pipelineresources.tekton.dev] (in: C:\Users\blubb\Downloads)
...Executing: kubectl [wait --for condition=established --timeout 300s crd/pipelineruns.tekton.dev] (in: C:\Users\blubb\Downloads)
.Executing: kubectl [wait --for condition=established --timeout 300s crd/pipelines.tekton.dev] (in: C:\Users\blubb\Downloads)
.Executing: kubectl [wait --for condition=established --timeout 300s crd/runs.tekton.dev] (in: C:\Users\blubb\Downloads)
.Executing: kubectl [wait --for condition=established --timeout 300s crd/taskruns.tekton.dev] (in: C:\Users\blubb\Downloads)
..Executing: kubectl [wait --for condition=established --timeout 300s crd/tasks.tekton.dev] (in: C:\Users\blubb\Downloads)
...Executing: kubectl [apply --filename C:\Users\blubb\AppData\Local\Temp\epinio073163558] (in: C:\Users\blubb\Downloads)
...........Executing: kubectl [get secret --namespace epinio-registry epinio-registry-tls -o jsonpath={.data.tls\.crt}] (in: C:\Users\blubb\Downloads)
..................Executing: kubectl [get secret --namespace epinio-registry epinio-registry-tls -o jsonpath={.data.tls\.crt}] (in: C:\Users\blubb\Downloads)
..
Ô£ö´©Å  CertManager deployed
......Executing: helm [install epinio-registry --namespace epinio-registry C:\Users\blubb\AppData\Local\Temp\epinio808585855 --set auth.htpasswd=f2f1ab2e6a0827d8:$2a$10$9JuEBtw2MV4P4WnDdoaEduDowS1ogJ6wjvUeb5kpfOnjIeDCO2f7K --set domain=epinio-registry.172.20.140.55.omg.howdoi.website --set createNodePort=true] (in: C:\Users\blubb\Downloads)
..........Executing: helm [install dex --values C:\Users\blubb\AppData\Local\Temp\epinio684788552 --namespace dex C:\Users\blubb\AppData\Local\Temp\epinio622106701] (in: C:\Users\blubb\Downloads)
.....Executing: kubectl [get secret --namespace epinio-registry epinio-registry-tls -o jsonpath={.data.tls\.crt}] (in: C:\Users\blubb\Downloads)
.................Executing: kubectl [get secret --namespace epinio-registry epinio-registry-tls -o jsonpath={.data.tls\.crt}] (in: C:\Users\blubb\Downloads)
.................Executing: kubectl [get secret --namespace epinio-registry epinio-registry-tls -o jsonpath={.data.tls\.crt}] (in: C:\Users\blubb\Downloads)
....
Ô£ö´©Å  ServiceCatalog deployed
..........Executing: kubectl [get secret --namespace epinio-registry epinio-registry-tls -o jsonpath={.data.tls\.crt}] (in: C:\Users\blubb\Downloads)
...........
Ô£ö´©Å  Registry deployed
.
Ô£ö´©Å  Dex deployed
Executing: kubectl [get secret --namespace epinio-registry epinio-registry-tls -o jsonpath={.data.tls\.crt}] (in: C:\Users\blubb\Downloads)
......Executing: kubectl [get secret --namespace epinio-registry epinio-registry-tls -o jsonpath={.data.tls\.crt}] (in: C:\Users\blubb\Downloads)
.....
Ô£ö´©Å  Gitea deployed
Executing: kubectl [get secret --namespace epinio-registry epinio-registry-tls -o jsonpath={.data.tls\.crt}] (in: C:\Users\blubb\Downloads)
....Executing: kubectl [get secret --namespace epinio-registry epinio-registry-tls -o jsonpath={.data.tls\.crt}] (in: C:\Users\blubb\Downloads)
...Executing: kubectl [get secret --namespace epinio-registry epinio-registry-tls -o jsonpath={.data.tls\.crt}] (in: C:\Users\blubb\Downloads)
...Executing: kubectl [get secret --namespace epinio-registry epinio-registry-tls -o jsonpath={.data.tls\.crt}] (in: C:\Users\blubb\Downloads)
...Executing: kubectl [get secret --namespace epinio-registry epinio-registry-tls -o jsonpath={.data.tls\.crt}] (in: C:\Users\blubb\Downloads)
...Executing: kubectl [get secret --namespace epinio-registry epinio-registry-tls -o jsonpath={.data.tls\.crt}] (in: C:\Users\blubb\Downloads)
...Executing: kubectl [get secret --namespace epinio-registry epinio-registry-tls -o jsonpath={.data.tls\.crt}] (in: C:\Users\blubb\Downloads)
...Executing: kubectl [get secret --namespace epinio-registry epinio-registry-tls -o jsonpath={.data.tls\.crt}] (in: C:\Users\blubb\Downloads)
...Executing: kubectl [get secret --namespace epinio-registry epinio-registry-tls -o jsonpath={.data.tls\.crt}] (in: C:\Users\blubb\Downloads)
...Executing: kubectl [get secret --namespace epinio-registry epinio-registry-tls -o jsonpath={.data.tls\.crt}] (in: C:\Users\blubb\Downloads)
....Executing: kubectl [get secret --namespace epinio-registry epinio-registry-tls -o jsonpath={.data.tls\.crt}] (in: C:\Users\blubb\Downloads)
...Executing: kubectl [get secret --namespace epinio-registry epinio-registry-tls -o jsonpath={.data.tls\.crt}] (in: C:\Users\blubb\Downloads)
...Executing: kubectl [get secret --namespace epinio-registry epinio-registry-tls -o jsonpath={.data.tls\.crt}] (in: C:\Users\blubb\Downloads)
...Executing: kubectl [get secret --namespace epinio-registry epinio-registry-tls -o jsonpath={.data.tls\.crt}] (in: C:\Users\blubb\Downloads)
.
Ô£ö´©Å  Tekton deployed
.
```

**Comment by kkaempf** [comment]
Epinio was run with `$trace = epinio install` in PowerShell. "Debug" output is what appeared on stdout, "Trace" output is the content of the `$trace` variable.

**Comment by kkaempf** [comment]
Fixed in #675
